packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -775,31 +774,32 @@ export default class TabDisks extends Component {
       )
   )
 
-  individualActions = [
-    ...(process.env.XOA_PLAN > 1
-      ? [
-          {
-            handler: vbd => exportVdi(vbd.vdi),
-            icon: 'export',
-            label: _('exportVdi'),
-          },
-          {
-            disabled: vbd => vbd.attached,
-            handler: vbd => importVdi(vbd.vdi),
-            icon: 'import',
-            label: _('importVdi'),
-          },
-        ]
-      : []),
+  actions = [
     {
-      handler: vbd => this._migrateVdi(vbd.vdi),
-      icon: 'vdi-migrate',
-      label: _('vdiMigrate'),
+      disabled: selectedVbds => some(selectedVbds, 'attached'),
+      handler: deleteVbds,
+      individualDisabled: vbd => vbd.attached,
+      individualHandler: deleteVbd,
+      icon: 'vdi-forget',
+      label: _('vdiForget'),
+      level: 'danger',
     },
     {
-      handler: vbd => copy(vbd.vdi.uuid),
-      icon: 'clipboard',
-      label: vbd => _('copyUuid', { uuid: vbd.vdi.uuid }),
+      disabled: selectedVbds => some(selectedVbds, 'attached'),
+      handler: selectedVbds => deleteVdis(uniq(map(selectedVbds, 'vdi'))),
+      individualDisabled: vbd => vbd.attached,
+      individualHandler: vbd => deleteVdi(vbd.vdi),
+      individualLabel: _('vdiRemove'),
+      icon: 'vdi-remove',
+      label: _('deleteSelectedVdis'),
+      level: 'danger',
+    },
+    {
+      handler: selectedVbds =>
+        this._migrateVdis(uniq(map(selectedVbds, 'vdi'))),
```

Comment:
What about snapshots and base copies? Should we filter them out?

Reply:
The collection, which passed to the `SortedTable`, contains the VBDs attached to the active VDIs of the VM.